Forge.Forms/src/Forge.Forms.Tests/BoundExpressionTests.cs

```diff
@@ -388,6 +388,26 @@ public void TestValueConverters()
             Assert.IsNull(expression.StringFormat);
         }
 
+        [TestMethod]
+        public void TestMultiConverters()
+        {
+            var context = new DummyFormContext(new DummyForm
+            {
+                Value = new Model
+                {
+                    Value = 5,
+                },
+                Context = new List<int> { 1, 2, 3, 4, 5 }
+            });
+
+            var expression =
+                BoundExpression.Parse(
+                    "{MultiBinding {Binding Value} {Binding Value} |Multiply:0.000}");
```

Comment:
I wonder what happens if the bindings inside have string formats. Are they ignored? Do we need to hoist them up? Are they even necessary? I guess string formats won't matter because for strings we have a custom and easier format.